tutorial/part-1-creating-views/README.md

```diff
@@ -0,0 +1,18 @@
+# Part 1: Creating a basic view
+
+Let’s learn by building a small app.  
+
+
+Through this tutorial, we’ll walk you through the creation of a basic Grocery Catalog application for an online grocery store called Oakry. The application is a multi-page dashboard used by Oakry’s Catalogue Managers to:
```

Comment:
These files seem redundant. Were they added incorrectly? Can you delete them?

Reply:
Lots of duplicates of files, can you go through this PR and remove the ones which are not to be used?

Reply:
I wonder how they're getting generated because I directly edited in gitbook. I'm unsure what happens when I delete them from git. Have any clue?